src/components/domain/modals/TaskCardEditModal.tsx

```diff
@@ -0,0 +1,261 @@
+import CommonButton from '@/components/common/commonbutton/CommonButton'
+import Image from 'next/image'
+import Input from '@/components/common/input'
+import Tag from '@/components/common/tag/Tag'
+import type { TagColor } from '@/types/common/tag'
+import { useState } from 'react'
+
+const TAG_COLORS: TagColor[] = [
+  'tag-orange',
+  'tag-pink',
+  'tag-blue',
+  'tag-green',
+  'tag-purple',
+  'tag-yellow',
+  'tag-red',
+  'tag-teal',
+  'tag-brown',
+  'tag-gray',
+]
+
+interface TaskCardEditModalProps {
+  cardInfo: {
+    status: string
+    assignee: string
+    title: string
+    description: string
+    dueDate: string
+    tags: { label: string; color: TagColor }[]
+    imageUrl: string
+  }
+}
+
+export default function TaskCardEditModal({
+  cardInfo,
+}: TaskCardEditModalProps) {
+  const [statusValue, setStatusValue] = useState(cardInfo.status)
+  const [selectValue, setSelectValue] = useState(cardInfo.assignee)
+  const [title, setTitle] = useState(cardInfo.title)
+  const [description, setDescription] = useState(cardInfo.description)
+  const [inputValue, setInputValue] = useState('')
+  const [tags, setTags] = useState(cardInfo.tags)
+  const [availableColors, setAvailableColors] = useState<TagColor[]>([
+    ...TAG_COLORS,
+  ])
+
+  const statusSelectChangeHandler = (
+    e: React.ChangeEvent<HTMLSelectElement>
+  ) => {
+    setStatusValue(e.target.value)
+  }
+
+  const selectChangeHandler = (e: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
네이밍 관련 의견입니다!
아래 `handleTagKeyDown`처럼 접두어는 동사로 넣는게 좋을것 같아요.
```suggestion
  const handleChangeStatusSelect = (
    e: React.ChangeEvent<HTMLSelectElement>
  ) => {
    setStatusValue(e.target.value)
  }

  const handleChangeSelect = (e: React.ChangeEvent<HTMLSelectElement>) => {
```